README.md

```diff
@@ -49,6 +49,21 @@ Add `pod 'Yams'` to your `Podfile`.
 
 Add `github "jpsim/Yams"` to your `Cartfile`.
 
+### Bazel
+
+In your WORKSPACE file
+
+```WORKSPACE
+YAMS_GIT_SHA = "SOME_SHA"
+http_archive(
+    name = "yams",
```

Comment:
Is it a bazelism to lowercase this name? I tend to capitalize Yams when referring to the project.

Reply:
I think it should be:

Reply:
I think it should be: `com_github_jpsim_yams`.

Reply:
We use `Yams`, so the target name can be referred to using the shorten name `@Yams` in `deps` (but we put the target at the top level).

Reply:
I think this should be up to the user, unless we need to hardcode it somewhere in the rules/targets.